openmls/src/schedule/mod.rs

```diff
@@ -745,6 +757,57 @@ impl ExporterSecret {
     }
 }
 
+/// A secret that we can derive secrets from, that are used outside of OpenMLS.
+/// In contrast to `[ExporterSecret]`, the `[ApplicationExportSecret]` is not
+/// persisted. It can be deleted after use to achieve forward secrecy.
+#[cfg(feature = "extensions-draft")]
+#[derive(Debug, Serialize, Deserialize, Clone)]
+#[cfg_attr(any(test, feature = "test-utils"), derive(PartialEq))]
+pub struct ApplicationExportSecret {
+    secret: Secret,
+}
+
+#[cfg(feature = "extensions-draft")]
+impl ApplicationExportSecret {
```

Comment:
The draft appears to have changed. Shall we implement what's in the draft now? This also goes back to numbering the feature with the draft number.
Adding a link here with the comments to where this is defined would also be good. Now that not everything is from rfc 9420 anymore.